src/main/java/mod/chiselsandbits/core/api/ChiselAndBitsAPI.java

```diff
@@ -322,4 +330,22 @@ public void endUndoGroup(
 		UndoTracker.getInstance().endGroup( player );
 	}
 
+	@Override
+	@SideOnly( Side.CLIENT )
+	public KeyBinding getKeyBinding(
+			ModKeyBinding modKeyBinding )
+	{
+		switch ( modKeyBinding.type )
```

Comment:
Possibly, at least that would break properly at compile time, if being a bit more verbose code wise.